include/onnxruntime/core/session/onnxruntime_cxx_inline.h

```diff
@@ -929,6 +929,22 @@ inline void SessionImpl<T>::Run(const RunOptions& run_options, const IoBinding&
   ThrowOnError(GetApi().RunWithBinding(this->p_, run_options, io_binding));
 }
 
+inline void CallbackBridge(void* user_data, OrtValue** outputs, size_t num_outputs, OrtStatusPtr status) {
+  RunAsyncCallbackStdFn* callback = reinterpret_cast<RunAsyncCallbackStdFn*>(user_data);
+  std::vector<Ort::Value> output_values;
```

Comment:
InlinedVector. Also use the # of outputs during construction.

Reply:
Are we going to expose the inline vector in the cxx header?

Reply:
Sure, but we can do ```std::vector<Ort::Value> output_values(num_outputs);```

Reply:
Problem is, Ort::Value has explicit default constructor ...
Even when it doesn't, later we will have to invoke operator = on some right values.
Let's simply do some reserve() and emplace_back().